fe/fe-core/src/main/java/com/starrocks/sql/optimizer/base/HashDistributionSpec.java

```diff
@@ -169,6 +171,41 @@ public EquivalentDescriptor getEquivDesc() {
         return equivDesc;
     }
 
+    public boolean canColocate(HashDistributionSpec o) {
+        if (!hashDistributionDesc.canColocate(o.hashDistributionDesc)) {
+            return false;
+        }
+        if (!hashDistributionDesc.isBucketLocal()) {
```

Comment:
move to HashDistributionDesc.canColocate

Reply:
talked and decided to refactor it later.